pkg/ccl/backupccl/key_rewriter_test.go

```diff
@@ -74,7 +74,7 @@ func TestKeyRewriter(t *testing.T) {
 		},
 	}
 
-	kr, err := makeKeyRewriterFromRekeys(keys.SystemSQLCodec, rekeys)
+	kr, err := makeKeyRewriterFromRekeys(keys.SystemSQLCodec, rekeys, nil)
```

Comment:
should I, should I not
nit: `nil /*...forget it

Reply:
Done, unfortunately.